modules/api-explorer/package.json

```diff
@@ -25,7 +25,7 @@
     "@phenyl/lambda-adapter": "~1.0.0",
     "@phenyl/memory-db": "~1.0.0",
     "@phenyl/rest-api": "~1.0.0",
-    "@phenyl/standards": "~1.0.0"
+    "phenyl-standards": "~0.2.9"
```

Comment:
phenyl-standards should be changed to `@phenyl/standards` now. please rebase from the latest typescript branch